pkgs/tools/admin/mycli/default.nix

```diff
@@ -28,15 +28,22 @@ buildPythonApplication rec {
       --ignore=mycli/packages/paramiko_stub/__init__.py
   '';
 
-  meta = {
+  # TODO: remove with next release
+  patches = [ ./0001-Remove-unnecessary-use-of-python-2-compat-type-900.patch ];
```

Comment:
Can you use fetchpatch and add add above which version is the next version?

Reply:
I tried using fetchpatch with the URL https://github.com/dbcli/mycli/commit/c89e69ae47c46fda761794d4e986fa694ab56ea1.patch but the patch does not apply cleanly on top of v1.22.2 because the changelog and authors file where modified since:

```
applying patch /nix/store/4y9pq7sq9dsgxm1268jzqk8zy7vp5cyy-c89e69ae47c46fda761794d4e986fa694ab56ea1.patch
patching file changelog.md
Hunk #1 FAILED at 6.
Hunk #2 FAILED at 785.
2 out of 2 hunks FAILED -- saving rejects to file changelog.md.rej
patching file mycli/AUTHORS
Hunk #1 FAILED at 75.
1 out of 1 hunk FAILED -- saving rejects to file mycli/AUTHORS.rej
patching file mycli/packages/completion_engine.py
```

But now i see that fetchpatch has a "excludes"-option - will use that now.